css/index.css

```diff
@@ -20,12 +20,17 @@ header {
   border-bottom: 1px solid #DDDDDD;
 }
 
-nav {
+header nav {
   position: absolute;
   right: 10px;
   top: 10px;
 }
 
+label[for="search"] {
+  display:block;
+  margin-bottom:.5em;
```

Comment:
changed to `px`. I'd say the main reason for `em` is design. My background is in Graphic Design. Print Designers have used `em` and `en` spaces since the advent of the press so that the negative space is consistent with whatever typeface is used, rather than an ambiguous unit.